test/verify/check-system-info

```diff
@@ -408,8 +413,15 @@ class TestSystemInfo(MachineCase):
         self.assertNotIn('BIOS', b.text('#hwinfo .info-table-ct'))
 
         # PCI should be shown
-        b.wait_in_text('#hwinfo .listing-ct caption', "PCI")
-        b.wait_in_text('#hwinfo .listing-ct', "0000:00:00.0")
+        b.wait_in_text(pci_selector + ' .listing-ct caption', "PCI")
+        b.wait_in_text(pci_selector + ' .listing-ct', "0000:00:00.0")
+
+        # Memory details should be shown
+        # demidecode not available on certain images
+        if m.image not in ['fedora-atomic']:
+            b.wait_in_text('#hwinfo #memory-listing .listing-ct caption', "Memory")
+            b.wait_in_text('#hwinfo #memory-listing .listing-ct', "DIMM 0")
+            b.wait_in_text('#hwinfo #memory-listing .listing-ct', "RAM")
```

Comment:
This is a bit thin -- in addition to this basic "use the system's dmidecode", the test could install a fake `dmidecode` with some static output, and verify that the parsing of all the properties works as expected. Similar to what happens with `lscpu` in test/verify/check-system-info.

Reply:
Hey Martin, 
I'll be needing help in this case as I don't understand what's being asked and what test cases to write for this type of scenario.